### PR TITLE
Docker log (Issue #2016)

### PR DESCRIPTION
**Pull request by lawliet1997**
###Rawgit Link-

https://rawgit.com/lawliet1997/lawliet1997.github.io/branch_1/#!pages/vi/vi-planet-installation-and-configuration.md

### Checklist

- [x] Added the Screenshot of the log when to press CTRL+C

### CODE REVIEW

File: pages/vi/vi-planet-installation-and-configuration.md [hunk 1]
```diff
@@ -142,7 +144,7 @@ Go to your community `planet` at <http://127.0.0.1>. You will be asked to:
     - Select `Community` from `Select Nation/Community`drop-down
     - Select `vi` from `Nation(s)` drop-down
     - Use your GitHub username as `Name` so we can locate you on Virtual Intern nation. (`Name` must be unique on the nation side, if you have configured `planet` using your GitHub username before, please add a letter or digits to the end of your original username.)
-    - `Code` is not editable and it will be automatically filled
+    - `Code` is not editabl e and it will be automatically filled
```
Comment (lmmrssa) [inline pages/vi/vi-planet-installation-and-configuration.md]: This should not have been changes so change it back

File: pages/vi/vi-planet-installation-and-configuration.md [hunk 2]
```diff
@@ -128,7 +128,9 @@ Follow the official [Install Docker Compose](https://docs.docker.com/compose/ins
   3c2309e92dc6        treehouses/couchdb:2.1.1    "tini -- /docker-ent…"   39 seconds ago      Up 48 seconds               4369/tcp, 9100/tcp, 0.0.0.0:2200->5984/tcp   planet_couchdb_1
   ```
 
-1. See log in action with `docker-compose -f planet.yml -p planet logs -f`, press 'CTRL+C' to exit logs view
+1. See log in action with `docker-compose -f planet.yml -p planet logs -f`, press 'CTRL+C' only after the you see the contents in the below screenshot to exit logs view
```
Comment (lmmrssa) [inline pages/vi/vi-planet-installation-and-configuration.md]: change this line to 
> 1. See log in action with `docker-compose -f planet.yml -p planet logs -f` until you see `planet_db-init_1 exited with code 0` as shown on image below

File: pages/vi/vi-planet-installation-and-configuration.md [hunk 3]
```diff
@@ -128,7 +128,9 @@ Follow the official [Install Docker Compose](https://docs.docker.com/compose/ins
...
+
+![docker log](images/Issue1.png)
```
Comment (lmmrssa) [inline pages/vi/vi-planet-installation-and-configuration.md]: Add line
> Press `CTRL+C` to exist logs view.

**Comment by lawliet1997** [comment]
Done..
please recheck it ..

**Comment by lmmrssa** [comment]
@lawliet1997 Your commits does not address any of requested change.

**Pull request closed without merging**